app/products/calls/utils.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Dictionary} from '@mm-redux/types/utilities';
+import {CallParticipant} from '@mmproducts/calls/store/types/calls';
+
+export function sortParticipants(participants?: Dictionary<CallParticipant>): CallParticipant[] {
```

Comment:
Not sure exactly how much more work it would be but to be 100% consistent with web we should also do an alphasort as first pass and have the user sharing their screen (if present) on top.

https://github.com/mattermost/mattermost-plugin-calls/blob/e2ffece5808ff85a20ebd724c77cd8903322653a/webapp/src/components/expanded_view/index.ts#L23-L25

Reply:
@streamer45 Yep, done! It's not a 100% consistent, since I'm using the displayname code used by mobile, which is a bit more complicated than the code used by the Calls plugin, but I think that's the right choice (keep mobile's name display consistent with mobile).

Reply:
@streamer45 Hah, forgot to push it -- sorry, you should be able to see it now.

Reply:
Thanks @cpoile . What about the presenter (the user currently sharing their screen) on top? We can defer it if it's a lot of work, let me know.

Reply:
Good catch -- I've added the presenter icon, made it appear first in the list, and tweaked the ui to make it look better (the hand/microphone/screen icons are now a bit smaller when in screen sharing mode because the avatars are smaller). I removed a bit of the padding from the header to give more space the the other rows. 
Looks like: 
<img width="919" alt="image" src="https://user-images.githubusercontent.com/1490756/160181424-591f8832-c388-4edb-97f9-d5bc8a790ada.png">